src/utils/environmentalVariables.ts

```diff
@@ -21,6 +21,8 @@ export const isDevelopment: () => boolean = () =>
 export const isProduction = (): boolean =>
   process.env.NODE_ENV === "production";
 
+export const isBeta = (): boolean => process.env.REACT_ENV_BETA === "true";
```

Comment:
Assuming it's a limitation of environment variables that they're all represented as strings? Since this would make more sense as a boolean.

Reply:
yup env variables are always strings.